<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingTab</class>
 <widget class="QWidget" name="SettingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="tournSaveLocation_2">
     <property name="text">
      <string>Tournament Save Location</string>
     </property>
     <property name="buddy">
      <cstring>tournSaveLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="tournSaveLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="tournSaveLocationButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="deckCountLabel">
     <property name="text">
      <string>Default Minimum Deck Count</string>
     </property>
     <property name="buddy">
      <cstring>minDeckCountSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="minDeckCountSpinBox">
     <property name="suffix">
      <string> Decks</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="matchSizeLabel">
     <property name="text">
      <string>Default Match Size</string>
     </property>
     <property name="buddy">
      <cstring>matchSizeSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="matchSizeSpinBox">
     <property name="suffix">
      <string> Players</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="drawPointsLabel">
     <property name="text">
      <string>Default Draw Points</string>
     </property>
     <property name="buddy">
      <cstring>drawPointsSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="drawPointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="byePointsLabel">
     <property name="text">
      <string>Default Bye Points</string>
     </property>
     <property name="buddy">
      <cstring>byePointsSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="byePointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lossPointsLabel">
     <property name="text">
      <string>Default Loss Points</string>
     </property>
     <property name="buddy">
      <cstring>lossPointsSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="lossPointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="winPointsLabel">
     <property name="text">
      <string>Default Win Points</string>
     </property>
     <property name="buddy">
      <cstring>winPointsSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="winPointsSpinBox">
     <property name="suffix">
      <string> Points</string>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QRadioButton" name="swissRadio">
     <property name="text">
      <string>Swiss</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QRadioButton" name="fluidRadio">
     <property name="text">
      <string>Fluid</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default Tournament Type</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="maxDeckCountSpinBox">
     <property name="suffix">
      <string> Decks</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Default Maximum Deck Count</string>
     </property>
     <property name="buddy">
      <cstring>maxDeckCountSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="reportCrashesEdit">
     <property name="text">
      <string>Report Crashes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
